monte_carlo.py

```diff
@@ -112,13 +120,14 @@ def transport_neutron(mat, bounds, tallies, fission_banks, first_round, mesh):
                 
             # r is a variable that contains the distance along the
             # direction vector to the boundary being tested  
-            
+
             # test each boundary
             for axis in axes:
                 for side in ['min', 'max']:
                     r = distance_to_cell_edge[axis][side] \
                             / neutron.get_direction(axis)
-                    if (r > 0 and r < tempd and r != -0.0):
+                    
+                    if (r > 0 and r < tempd):
```

Comment:
The check 'r != 0.0' was needed when the first check was 'r >= 0'. I'm not sure why, but it was needed. Somewhere along the line, I changed the 'r >= 0' to 'r > 0' and just now saw that the third check was no longer needed.